sql/core/src/main/scala/org/apache/spark/sql/execution/streaming/state/HDFSBackedStateStoreProvider.scala

```diff
@@ -204,7 +208,15 @@ private[sql] class HDFSBackedStateStoreProvider extends StateStoreProvider with
     override def removeColFamilyIfExists(colFamilyName: String): Unit = {
       throw StateStoreErrors.removingColumnFamiliesNotSupported(
         "HDFSBackedStateStoreProvider")
+    }
+
+    override def valuesIterator(key: UnsafeRow, colFamilyName: String): Iterator[UnsafeRow] = {
+      throw StateStoreErrors.unsupportedOperationException("multipleValuesPerKey", "HDFSStateStore")
+    }
 
+    override def merge(key: UnsafeRow, value: UnsafeRow,
```

Comment:
Nit: lets move all arguments to new line with required spacing ?

Reply:
Done.

Reply:
Done